pint_pandas/testsuite/test_pandas_extensiontests.py

```diff
@@ -287,18 +287,27 @@ def test_in_numeric_groupby(self, data_for_grouping):
                 "C": [1, 1, 1, 1, 1, 1, 1, 1],
             }
         )
-        result = df.groupby("A").sum().columns
-
-        # FIXME: Why dies C get included for e.g. PandasDtype('complex128') but not for Float64Dtype()? This seems buggy,
-        # but very hard for us to fix...
-        if df.B.isna().sum() == 0 or isinstance(
-            df.B.values.data.dtype, pd.core.dtypes.dtypes.PandasDtype
-        ):
-            expected = pd.Index(["B", "C"])
-        else:
-            expected = pd.Index(["C"])
+        result = df.groupby("A").sum()
```

Comment:
#172 didn't need as much changing here

Reply:
The new code would be fine with the same test as #172, ie for all cases expect there to be the columns `"B"` and `"C"`. But seriously that is just a bad test.

Reply:
it's a test that's inherited from pandas to ensure ExtensionArrays behave as expected; we should be changing these tests as little as possible

it would be really helpful to have more tests for how pint-pandas is used. df.sum(), max and so on aren't working but there's no test catching these at the moment